docs/TagSeqDiagram/ExecuteTagOp.puml

```diff
@@ -0,0 +1,34 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+skinparam shadowing false
+participant ":TagCommand" as TagCommand
+
+mainframe **sd** Execute Tag Operation
+
+activate TagCommand
+
+alt TagOp == add
+TagCommand -> TagCommand:addTag(targetModule:Module)
+activate TagCommand
+TagCommand ->] :getTaskList()
+TagCommand <-] ::TaskList
+TagCommand ->] :addTag(tagDescription:String, taskIndex:int)
+TagCommand <-] ::Task
+return result
+
+else TagOp == del
+TagCommand -> TagCommand:removeTag(targetModule:Module))
+activate TagCommand
+TagCommand ->] :getTaskList()
+TagCommand <-] ::TaskList
+TagCommand ->] :delTag(tagDescription:String, taskIndex:int)
+TagCommand <-] ::Task
+return result
+
+else else
+[<-- TagCommand:e:ParseException
+
+end
```

Comment:
The TagCommand will return a CommandResult Object, so should call the constructor of CommandResult before end?

Reply:
Missed that, thanks for pointing it out